src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExperimentalOptions.cs

```diff
@@ -27,10 +29,20 @@ public ExperimentalOptions(IConfiguration configuration)
         {
             this.EmitLogEventAttributes = emitLogEventAttributes;
         }
+
+        if (configuration.TryGetBoolValue(EnableInMemoryRetryEnvVar, out var enableInMemoryRetry))
+        {
+            this.EnableInMemoryRetry = enableInMemoryRetry;
+        }
     }
 
     /// <summary>
-    /// Gets or sets a value indicating whether log event attributes should be exported.
+    /// Gets a value indicating whether log event attributes should be exported.
+    /// </summary>
+    public bool EmitLogEventAttributes { get; }
+
+    /// <summary>
+    /// Gets a value indicating whether retries should be enabled in case of transient errors.
```

Comment:
please add a link to the spec being followed to determine transient or not.
like https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/specification.md